perfkitbenchmarker/cloudstack/cloudstack_virtual_machine.py

```diff
@@ -188,13 +189,14 @@ def CreateScratchDisk(self, disk_spec):
 
     for i in xrange(disk_spec.num_striped_disks):
 
-        name = 'disk-%s-%s' % (self.name, i + 1)
+        name = 'disk-%s-%d-%d' % (self.name, i + 1, self.disk_counter)
```

Comment:
I'm not sure I follow this  - when will `self.name` and `i` not uniquely identify a disk?